runtime/js/30_metrics.js

```diff
@@ -4,7 +4,11 @@
   const core = window.Deno.core;
 
   function metrics() {
-    return core.jsonOpSync("op_metrics");
+    const { combined, ops } = core.jsonOpSync("op_metrics");
+    if (ops) {
+      combined.ops = ops;
+    }
```

Comment:
When is this condition false?

Reply:
If unstable is not enabled.

Reply:
So the interface will change once again, once we stabilize this feature? (presumably for 2.0 as it would be a breaking change)

Reply:
Indeed. That was my thinking.